subiquity/ui/views/filesystem/partition.py

```diff
@@ -208,6 +208,13 @@ def validate_mount(self):
     'installed onto this partition, which must be formatted as fat32. The '
     'only aspect of this partition that can be edited is the size.')
 
+prep_partition_description = _(
+    "Required bootloader partition\n"
+    "\n"
+    'This is the PReP partion which is required on POWER systems. Grub will be '
+    'installed onto this partition. The '
+    'only aspect of this partition that can be edited is the size.')
```

Comment:
I don't think the size should be editable; or at least we should require a minimum size.  I've seen recommendations for 8, 10, 16 and 32MB in size.

Reply:
Yeah, I have no idea what I am doing here. @adconrad is PReP partition always 8MiB? Fixed size? Allow users to twiddle it or not?